src/Factory/Api/Mastodon/Notification.php

```diff
@@ -24,23 +24,18 @@
 use Friendica\BaseFactory;
 use Friendica\Database\DBA;
 use Friendica\DI;
-use Friendica\Model\Contact;
-use Friendica\Model\Notification as ModelNotification;
+use Friendica\Model\Post;
+use Friendica\Model\Verb;
+use Friendica\Protocol\Activity;
 
 class Notification extends BaseFactory
 {
 	public function createFromNotifyId(int $id)
```

Comment:
Please rename to match the new table's name.
```suggestion
	public function createFromNotificationId(int $id)
```

Reply:
Done